docs/book/src/crd/index.md

```diff
@@ -1244,6 +1244,17 @@ create S3 Buckets for workload clusters.
 TODO: This field could be a pointer, but it seems it breaks setting default values?</p>
 </td>
 </tr>
+<tr>
```

Comment:
What prompted this change? 🤔

Reply:
I ran make docs.  If this is not supposed to be done, I can revert the file.

Reply:
I have no idea I never ran the thing. 🤣🤣🤣

Reply:
removed then :)